tests/lib/rules/no-unused-vars.js

```diff
@@ -376,6 +376,14 @@ ruleTester.run("no-unused-vars", ruleNoUnusedVars, {
         {
             code: [
                 "import { Nullable } from 'nullable'",
+                "const foo = ({ nullable }: Nullable)  => nullable",
```

Comment:
`Nullable) => Nullable`? (remove the space; capitalize `nullable`)

Reply:
That won't be what I meant.

`const foo = ` is assignment. `(` starts arrow function. `{ nullable }` is destructuring assignment. `: Nullable` is type annotation for the value before destructuring. The rest is clear. Although I do have an extra space before the arrow (negligible?).

Reply:
@j-f1 I think you are looking at this as a declaration, whilst @mightyiam defined as an actual arrow function... there is one more thing I noticed though.. the last call foo(); won't compile in typescript as an argument is required by the arrow function...

Reply:
Also, the extra whitespace won't really matter, but if you think about this... we are developing a plugin for a linting tool... it would by kind of ironic not to have our own code linted... what do you think?